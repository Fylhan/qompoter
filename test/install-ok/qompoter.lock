{
  "name": "install-ok/qompoter",
  "date": "2016-11-24T22:01:22+01:00",
  "require": {
    "cp/acme-cp": {"version": "v1.0", "type": "qompoter-fs", "url": "", "md5sum": "115f3cd6f4a5dec31fb31b9cf39fa855"},
    "cp/fylhan2": {"version": "v1.0", "type": "qompoter-fs", "url": "", "md5sum": "274036013605b10c1f52a688080761dd"},
    "fylhan/platphorm": {"version": "master", "type": "git", "url": "", "md5sum": "f3825863afd8ecacdfa51d66bc5c4072"},
    "cp/fylhan": {"version": "v1.0", "type": "qompoter-fs", "url": "", "md5sum": "ea9f548ff879e721c12085d7cddce551"},
    "fylhan/posibrain": {"version": "v0.2", "type": "git", "url": "", "md5sum": "f2be7bd1e5da767a9a9b0c8dc3c7aa31"},
  }
}
