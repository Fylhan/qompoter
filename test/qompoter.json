{
	"name": "fylhan/qompoter-test",
	"description": "Test qompoter file",
	"keywords": ["Qt", "C++", "dependency management"],
	"license": "LGPL3+",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"trialog/solilog": "v1.0",
		"trialog/gpslib": "v1.1.1",
		"trialog/tcanp": "v1.6.7-lib",
		"ixxat/eci": "v1.8-lib"
	},
	"require-dev": {
		"trialog/autotester": "v1.0"
	},
	"repositories": [
		{
			"type": "fs",
			"url": "/media/data/Project/qompoter"
		}
	]
}
