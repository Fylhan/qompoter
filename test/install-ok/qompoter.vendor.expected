vendor
vendor/acme
vendor/acme/qompoter.json
vendor/acme/qompoter.pri
vendor/acme-tag
vendor/acme-tag/acme.txt
vendor/acme-tag/qompoter.json
vendor/acme-tag/qompoter.pri
vendor/fylhan
vendor/fylhan/qompoter.json
vendor/fylhan/qompoter.pri
vendor/qompote.pri
vendor/vendor.pri
